test/hotspot/jtreg/TEST.groups

```diff
@@ -377,8 +377,6 @@ tier1_runtime = \
  -runtime/memory/ReserveMemory.java \
  -runtime/Metaspace/FragmentMetaspace.java \
  -runtime/Metaspace/FragmentMetaspaceSimple.java \
- -runtime/Metaspace/elastic/TestMetaspaceAllocationMT1.java \
- -runtime/Metaspace/elastic/TestMetaspaceAllocationMT2.java \
```

Comment:
Reviewer hint: I need these tests back in GHA tier 1 for the moment (also because more Metaspace changes are coming for Lilliput)

Reply:
Yes, I have planned to make them faster, or cut them down a bit in automatic mode: https://bugs.openjdk.org/browse/JDK-8303815

Reply:
(Patch is up for review; I cut down the time they take significantly)